validator.py

```diff
@@ -83,5 +98,3 @@ def validate(file):
 
     print(f"Found {results.frame_count} frames, step time of {results.step_time} ms for a total duration of {datetime.timedelta(seconds=results.duration_s)}.")
```

Comment:
Let's add command count to non-error cases as well, for checking how many commands you've got left:

```suggestion
    print(f"Found {results.frame_count} frames and {results.command_count} commands, step time of {results.step_time} ms for a total duration of {datetime.timedelta(seconds=results.duration_s)}.")
```